emp/emp_role_cftemplate.yaml

```diff
@@ -0,0 +1,377 @@
+              - ec2:DeleteVpcEndpoints
+              - ec2:DescribeAccountAttributes
+              - ec2:DescribeAddresses
+              - ec2:DescribeAvailabilityZones
+              - ec2:DescribeCarrierGateways
+              - ec2:DescribeInstances
+#              - ec2:DescribeInstanceTypes commenting out capa actions, accounted for below
+              - ec2:DescribeInternetGateways
+              - ec2:DescribeEgressOnlyInternetGateways
+#              - ec2:DescribeInstanceTypes commenting out capa actions, accounted for below
+#              - ec2:DescribeImages commenting out capa actions, accounted for below
+              - ec2:DescribeNatGateways
+              - ec2:DescribeNetworkInterfaces
+              - ec2:DescribeNetworkInterfaceAttribute
+              - ec2:DescribeRouteTables
+              - ec2:DescribeSecurityGroups
+              - ec2:DescribeSubnets
+              - ec2:DescribeVpcs
+              - ec2:DescribeDhcpOptions
+              - ec2:DescribeVpcAttribute
+              - ec2:DescribeVpcEndpoints
+              - ec2:DescribeVolumes
+              - ec2:DescribeTags
+              - ec2:DetachInternetGateway
+              - ec2:DisassociateRouteTable
+              - ec2:DisassociateAddress
+              - ec2:ModifyInstanceAttribute
+              - ec2:ModifyNetworkInterfaceAttribute
+              - ec2:ModifySubnetAttribute
+              - ec2:ReleaseAddress
+              - ec2:RevokeSecurityGroupIngress
+              - ec2:RunInstances
+              - ec2:TerminateInstances
+              - tag:GetResources
+              - elasticloadbalancing:AddTags
+              - elasticloadbalancing:CreateLoadBalancer
+              - elasticloadbalancing:ConfigureHealthCheck
+              - elasticloadbalancing:DeleteLoadBalancer
+              - elasticloadbalancing:DeleteTargetGroup
+              - elasticloadbalancing:DescribeLoadBalancers
+              - elasticloadbalancing:DescribeLoadBalancerAttributes
+              - elasticloadbalancing:DescribeTargetGroups
+              - elasticloadbalancing:ApplySecurityGroupsToLoadBalancer
+              - elasticloadbalancing:DescribeTags
+              - elasticloadbalancing:ModifyLoadBalancerAttributes
+              - elasticloadbalancing:RegisterInstancesWithLoadBalancer
+              - elasticloadbalancing:DeregisterInstancesFromLoadBalancer
+              - elasticloadbalancing:RemoveTags
+              - elasticloadbalancing:SetSubnets
+              - elasticloadbalancing:ModifyTargetGroupAttributes
+              - elasticloadbalancing:CreateTargetGroup
+              - elasticloadbalancing:DescribeListeners
+              - elasticloadbalancing:CreateListener
+              - elasticloadbalancing:DescribeTargetHealth
+              - elasticloadbalancing:RegisterTargets
+              - elasticloadbalancing:DeleteListener
+              # commenting out capa actions, reason: we don't use machinepools, and just scale the md
```

Comment:
do we want to remove this commented part? as user will just download and apply the cft.

Reply:
why do we need this commented parts in public exposed cft.